views/confirmation-still-required.html

```diff
@@ -48,6 +48,15 @@ <h2 class="govuk-heading-m">What happens next</h2>
             but it must file its {{ overdueFiling }} to stay on the register.
         </p>
 
+        {# This block is displayed if ONLY the confirmation statement is late: #}
+        {% if (company.isAccountsOverdue === false) and (company.isConfirmationStatementOverdue === true) %}
+            <p>
+                If {{ company.companyName }} has not filed by {{ newDeadline }}, we may prosecute the directors of the company.
```

Comment:
Bit picky I know but why is this a "new deadline" if it's not been changed or extended?